draft/2022-08-10-this-week-in-rust.md

```diff
@@ -36,9 +36,8 @@ and just ask the editors to select the category.
 ### Project/Tooling Updates
 
 ### Observations/Thoughts
-
```

Comment:
Github is having a tough time resolving the merge conflict...  can this blank line be restored? Thank you!